Telegram/SourceFiles/info/profile/info_profile_cover.cpp

```diff
@@ -125,12 +125,12 @@ constexpr auto kGlareTimeout = crl::time(1000);
 }
 
 [[nodiscard]] MusicButtonData DocumentMusicButtonData(
-		not_null<DocumentData*> document) {
+	not_null<DocumentData*> document) {
 	if (const auto song = document->song()) {
 		if (!song->performer.isEmpty() || !song->title.isEmpty()) {
 			return {
 				.performer = song->performer,
-				.title = song->title,
+				.title = u"- "_q+song->title,
```

Comment:
```suggestion
				.title = u"- "_q + song->title,
```

Reply:
It looks like the "the “-” sign remains" part is not fixed? I suppose it has to be
```suggestion
				.title = (song->performer.isEmpty()
					? QString()
					: u"- "_q) + song->title,
```